doc/maintainer/manual-build.rst

```diff
@@ -247,3 +250,34 @@ Build GDB::
     $ make install-gdb
     $ cd -
 
+
+Glibc toolchain
+```````````````
+
+Glibc toolchain is builded like the uClibc. But there are following differences:
+
+1. Change `--target=arc-snps-linux-uclibc` to `--target=arc-snps-linux-gnu`.
+2. Replace uClibc stages to glibc stages:
```

Comment:
To be honest, I think, this list is hard to read, because commands are semantically parts of the second item in a list, but they don't look visually like this in text. I'd recomend removing the list formatting and instead do just a text.

```
Glibc toolchain is built like the uClibc toolchain, but there are few differences.
First, it is needed to change ``--target=arc-snps-linux-uclibc`` to ``--target=arc-snps-linux-gnu``.
Second, uClibc-specific stages should be replaced with following glibc-specific stages.
```